cognite/neat/rules/_importer/_spreadsheet2rules.py

```diff
@@ -0,0 +1,85 @@
+"""This module performs importing of graph to TransformationRules pydantic class.
+In more details, it traverses the graph and abstracts class and properties, basically
+generating a list of rules based on which nodes that form the graph are made.
+"""
+
+from pathlib import Path
+from typing import cast
+
+import pandas as pd
+
+from cognite.neat.rules.models._rules import RULES_PER_ROLE, DMSRules, DomainRules, InformationRules
+from cognite.neat.rules.models._rules.base import RoleTypes
+from cognite.neat.utils.auxiliary import local_import
+from cognite.neat.utils.spreadsheet import read_spreadsheet
+
+from ._base import BaseImporter
+
+
+class ExcelImporter(BaseImporter):
+    def __init__(self, filepath: Path):
+        self.filepath = filepath
+
+    def to_rules(self, role: RoleTypes | None = None, skiprows: int = 1) -> DomainRules | InformationRules | DMSRules:
```

Comment:
I think this need to be smarter, or at least have an option where you input None to skiprows and it searches for the headings. But that can be move to a later PR.